### PR TITLE
Buffs to basic medicines

### PR DESCRIPTION
**Pull request by EvilDragonfiend**
## About The Pull Request
Buffs to basic medicines
* metabolisation rate is now 0.1u (bicaridine, kelotane, antitoxin, dexalin)
* Tricord is 0.6u (because x6)
* reduces overdose penalty since metabolisation is slow and healing is weak

## Why It's Good For The Game
My nerfs to basic chems are pretty much working as intended, but it's true that it's awful even to treat non-severe damages.
I will make these slowly metabolised (like omnizine). It will be how it should exactly be - as long term treatment to heal minor damages

## Testing Photographs and Procedure

basically var edit, so not necessary.

## Changelog
:cl:
balance: Bicaridine/Kelotane/Antitoxin/Dexalin will be now metabolised by 0.1u
balance: Tricordrazine will be now metabolised by 0.6u
balance: Said medicines now punish people less when they're overdosed with those. (i.e. 2 Liver damage to 0.5 liver damage)
/:cl:

### CODE REVIEW

**Comment by EvilDragonfiend** [comment]
Need wiki changes, but I can do that immediately once it's merged

**Comment by Rukofamicom** [comment]
Working on my own adjustments to these (and white medkit items) as well that is closely intertwined. 

I just desperately need this merged before I can make a PR for it: #8507

**Review by PowerfulBacon — DISMISSED** [review]
Update the wiki and ping for mergre

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -1326,11 +1327,11 @@
 
 /datum/reagent/medicine/tricordrazine
 	name = "Tricordrazine"
-	description = "Has a high chance to heal all types of damage. Overdose causes toxin damage and liver damage."
+	description = "Heals all types of damage based on damage types you have and its severity. Overdose causes toxin damage and liver damage."
 	reagent_state = LIQUID
 	color = "#707A00" //tricord's component chems mixed together, olive.
 	chem_flags = CHEMICAL_RNG_GENERAL | CHEMICAL_RNG_FUN | CHEMICAL_RNG_BOTANY
-	metabolization_rate = 3 * REAGENTS_METABOLISM
+	metabolization_rate = 1.5 * REAGENTS_METABOLISM
```
Comment (PowerfulBacon) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm:1334]: I think considering tricordrazine is a little harder to make the healing should be better.
It has 1/3 the effectiveness when compared to the other basic chemicals due to the high metabolisation rate but still low healing amount.
I think a heal value of ~2.5 or a metabolisation rate of 0.6 would be an improvement, since then its only slightly worse than the basic chems but has the increased speed and healing all of them.

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]

**Comment by PowerfulBacon** [comment]
I think I agree with the metabolites PR in terms of method and impact. This PR conflicts with it

**Comment by PowerfulBacon** [comment]
Closing in favour of https://github.com/BeeStation/BeeStation-Hornet/pull/8572
